R/datasets.R

```diff
@@ -58,13 +63,22 @@ datasets.read_table <- function(alias, columns = NULL, row_limit = NULL, format
 
 #' Writes a data.frame to a Foundry dataset.
 #'
+#' @section Column types:
 #' Note that types may not be exactly preserved and all types are not supported.
 #' See https://arrow.apache.org/docs/r/articles/arrow.html for details on type conversions
 #' from a data.frame to an arrow Table. Use arrow::Table$create to use more granular types.
 #'
+#' @section Row Names:
+#' Row names are silently removed.
```

Comment:
What does this mean?

Reply:
R dataframes can have names for the index instead of numbers, but it's not recommended to use them anymore https://benwhalley.github.io/just-enough-r/rownames.html and most standard packages drop them by default now. Just clarifying the behavior so users aren't surprised